scripts/packages/VSCodeServer/src/repl.jl

```diff
@@ -92,22 +92,23 @@ function evalrepl(m, line, repl, main_mode)
 end
 
 function display_parse_error(io, err::Meta.ParseError)
-  printstyled(io, "ERROR: "; bold = true, color = Base.error_color())
-  printstyled(io, "syntax: ", err.msg; color = Base.error_color())
-  println(io)
+    printstyled(io, "ERROR: "; bold = true, color = Base.error_color())
+    printstyled(io, "syntax: ", err.msg; color = Base.error_color())
+    println(io)
```

Comment:
this function is actually a dead code, so let's remove this ?

Reply:
Done.